gufe/tests/conftest.py

```diff
@@ -67,7 +67,12 @@ def get_test_filename(filename):
     name: URLFileLike(url=_pl_benchmark_url_pattern.format(name=name)) for name in _benchmark_pdb_names
 }
 
-PDB_FILE_LOADERS = {name: lambda: get_test_filename(name) for name in ["181l.pdb"]}
+PDB_FILE_LOADERS = {
+    name: (lambda name=name: get_test_filename(name))
```

Comment:
this was fun https://docs.python.org/3/faq/programming.html#why-do-lambdas-defined-in-a-loop-with-different-values-all-return-the-same-result